job_creator.php

```diff
@@ -489,6 +489,9 @@ public function createJson(string $yml): string
 
         $this->githubRepository = $inputs['github_repository'];
         $this->installerVersion = $this->getInstallerVersion();
+        if ((bool) preg_match($unstableTagRx, $myRef)) {
```

Comment:
```suggestion
        if ((bool) preg_match($unstableTagRx, $myRef, $m)) {
```
I know we're capturing the matches in `$m` up above, but this safeguards against us changing that regex or logic in the future in a way that changes what `$m[3]` is by the time we get to this part of the code.

Reply:
Updated